add file filters for non-compilable files in openjdk16

Failure from https://github.com/checkstyle/checkstyle/runs/4851584619?check_suite_focus=true
Code-
```
@AnonSubclassOnPkg(new Object(){})
package pkg;
```
```
Caused by: java.lang.NullPointerException
    at com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheck.isInsideLocalAnonInnerClass (UnusedLocalVariableCheck.java:443)
    at com.puppycrawl.tools.checkstyle.checks.coding.UnusedLocalVariableCheck.visitToken (UnusedLocalVariableCheck.java:326)
    at com.puppycrawl.tools.checkstyle.TreeWalker.notifyVisit (TreeWalker.java:335)
    at com.puppycrawl.tools.checkstyle.TreeWalker.processIter (TreeWalker.java:406)
    at com.puppycrawl.tools.checkstyle.TreeWalker.walk (TreeWalker.java:273)
    at com.puppycrawl.tools.checkstyle.TreeWalker.processFiltered (TreeWalker.java:154)
    at com.puppycrawl.tools.checkstyle.api.AbstractFileSetCheck.process 
```
Exception is thrown for this file-
https://github.com/openjdk/jdk16/blob/master/test/langtools/tools/javac/annotations/neg/pkg/package-info.java
The code is noncompilable, an annotation attribute value must be a constant expression. This file is a test file for error messages.

--- a/.ci/openjdk16-excluded.files
+++ b/.ci/openjdk16-excluded.files
@@ -597,3 +597,9 @@
 <module name="BeforeExecutionExclusionFileFilter">
   <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]generics[\\/]6413682[\\/]T6413682.java$"/>
 </module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]annotations[\\/]neg[\\/]pkg[\\/]package-info.java$"/>
+</module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]annotations[\\/]neg[\\/]AnonSubclass.java$"/>
+</module>

